pinot-api/pom.xml

```diff
@@ -56,6 +56,12 @@
     <dependency>
       <groupId>com.ning</groupId>
       <artifactId>async-http-client</artifactId>
+      <exclusions>
```

Comment:
I think after excluding this dependency, you have to explicitly pull in a dependency or it might not work right?

Reply:
If we print out the dependency tree, we'll find out there's another same jar but with higher version in the same module.
```[INFO] com.linkedin.pinot:pinot-api:jar:0.016
[INFO] +- com.ning:async-http-client:jar:1.9.21:compile
[INFO] |  \- io.netty:netty:jar:3.10.1.Final:compile
[INFO] \- com.101tec:zkclient:jar:0.7:compile
[INFO]    \- org.apache.zookeeper:zookeeper:jar:3.4.9:compile (version managed from 3.4.6)
[INFO]       \- (io.netty:netty:jar:3.10.5.Final:compile - omitted for conflict with 3.10.1.Final)
```
But after excluding it from `com.ning:async-http-client`, the dependency tree now looks like:
```
[INFO] com.linkedin.pinot:pinot-api:jar:0.016
[INFO] \- com.101tec:zkclient:jar:0.7:compile
[INFO]    \- org.apache.zookeeper:zookeeper:jar:3.4.9:compile (version managed from 3.4.6)
[INFO]       \- io.netty:netty:jar:3.10.5.Final:compile
```
maven will traverse the tree and then find the  io.netty:netty:jar in deeper level and ultimately imports that `3.10.5.Final` jar, which is what we want.

But yeah, the strategy of explicitly putting dependency in higher level might be the better approach. I'll adjust the code. 👍